forms-flow-web/src/components/CustomComponents/FormInput.js

```diff
@@ -23,7 +23,6 @@ const FormInput = ({
 
   const inputClassNames = "form-control-input " + className ;
   return (
-    <div className="form-input-box" >
       <Form.Group controlId={id}>
       {label && (
```

Comment:
This component is removed from open source in last PR .

Reply:
Ok . will update the PR